recipe/meta.yaml

```diff
@@ -21,9 +21,10 @@ requirements:
     - python >=3.6
     - setuptools >=42
   run:
+    - dataclasses # [py==36]
```

Comment:
Pretty sure this is how Python version-specific deps work: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors